typescript/web/src/utils/export/csv-stringify-async.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  Options as StringifyCsvOptions,
+  stringify as stringifyCsv,
+} from "csv-stringify";
+
+export const stringifyCsvAsync = (
+  rows: unknown[][],
+  options?: StringifyCsvOptions
+): Promise<string> =>
+  new Promise((resolve, reject) => {
+    stringifyCsv(rows, options, (error, output) => {
+      if (error) {
+        reject(error);
+      } else {
+        resolve(output);
+      }
+    });
+  });
```

Comment:
The same code already exists in `common-resolvers/src/export/export-to-csv.ts`. Can you please merge both?